.github/workflows/eden-trusted.yml

```diff
@@ -174,11 +175,49 @@ jobs:
               }
             }
 
+  tests-13-4-stable:
```

Comment:
From what I see `tests-13-4-stable`, `tests-14-5-stable`, `tests-master` only differ in eden version, is it possible to use env variables, i.e. ${{eden_version}}} in uses tag? I believe you can, then we can just have action to determine eden version and then have generic test step that would use eden_version variable from previous step, that would imo have clearer flow and reduce finalize step

Reply:
no, sadly it's not possible. the string that comes after `uses:` needs to be static